protos/odpf/optimus/core/v1beta1/job_spec.swagger.json

```diff
@@ -88,6 +88,166 @@
             "in": "path",
             "required": true,
             "type": "string"
+          },
+          {
+            "name": "job.version",
```

Comment:
all these changes are new which ideally shouldn't change with buf version

Reply:
Should not be part of changes. Reverting changes for buf update, maybe some issue.